lib/puppet/provider/elasticsearch_plugin/plugin.rb

```diff
@@ -90,7 +98,12 @@ def destroy
   def es_version
     return @es_version if @es_version
     begin
-      version = es('-v') # ES 1.x
+      os = Facter['osfamily'].value
+      if os == 'OpenBSD'
+        version = pkg_info('elasticsearch').lines.first
```

Comment:
Ah. we don't look in /usr/local/elasticsearch/bin by default at https://github.com/elastic/elasticsearch/blob/master/distribution/src/main/resources/bin/elasticsearch#L79-L95

Would be nice to have that fixed in the actual binary as well.
For the time being you can indeed set the ENV variable before running the command.
At least that way the local stays the same.